syntax = "proto2";

message Empty {}

message Test {
    required string name = 1;
}

message saveTestResponse {
  required Test name = 1;
}

message getTestResponse {
  optional Test name = 1;
}

service TestService  {
  rpc SaveTesting(Test) returns (saveTestResponse) {}
  rpc GetTesting(Empty) returns (getTestResponse) {}
}